examples/07-EMIT/ComputeProtectionLevels.py

```diff
@@ -4,366 +4,376 @@
 This example shows how you can use PyAEDT to open an AEDT project with
 an EMIT design and analyze the results to determine if the received 
 power at the input to each receiver exceeds the specified protection
-levels. 
+levels.
+This example requires Ansys AEDT 2023 R2. Uncomment it and run on correct version.
```

Comment:
```suggestion
This example requires Ansys AEDT 2023 R2 or later. Uncomment it and run on correct version.
```
I know something later than AEDT 2023 R2 is many months away, but it avoids maintenance later.